examples/mlee.py

```diff
@@ -0,0 +1,610 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This is a template on how to implement a dataset in the biomedical repo.
+
+A thorough walkthrough on how to implement a dataset can be found here:
+https://huggingface.co/docs/datasets/add_dataset.html
+
+This script corresponds to Step 4 in the Biomedical Hackathon guide.
+
+To start, copy this template file and save it as <your_dataset_name>.py in an appropriately named folder within datasets. Then, modify this file as necessary to implement your own method of extracting, and generating examples for your dataset. 
+
+There are 3 key elements to implementing a dataset:
+
+(1) `_info`: Create a skeletal structure that describes what is in the dataset and the nature of the features.
+
+(2) `_split_generators`: Download and extract data for each split of the data (ex: train/dev/test)
+
+(3) `_generate_examples`: From downloaded + extracted data, process files for the data in a feature format specified in "info".
+
+----------------------
+Step 1: Declare imports
+Your imports go here; the only mandatory one is `datasets`, as methods and attributes from this library will be used throughout the script.
+
+We have provided some import statements that we strongly recommend. Feel free to adapt; so long as the style-guide requirements are satisfied (Step 5), then you should be able to push your code.
```

Comment:
For the examples I think we should remove this boilerplate and put a description of the dataset (following https://github.com/huggingface/datasets/blob/master/templates/new_dataset_script.py)

Reply:
Thanks for the review! The boilerplate comments were indeed misleading. I have replaced them with comments specific to MLEE which are hopefully more useful in 6525b94fea387d4fc0346e90f056c507313a393a.